core/platform/datastore/gae_datastore_services_test.py

```diff
@@ -99,6 +99,9 @@ class MockDatetimeForDatastoreTests(test_utils.GenericTestBase):
     def test_exception_is_raised_when_passed_wrong_type(self):
         # type: () -> None
         with self.assertRaisesRegexp(Exception, 'mocked_now must be datetime'): # type: ignore[no-untyped-call]
+            # TODO(#13528): Remove this test after back-end is fully type
+            # annotated. Here ignore[arg-type] is used to test method for
```

Comment:
Say which method (e.g. "test the XXX() method"). DItto for all similar comments.

Reply:
Done